### ISSUE TITLE
path import is broken 

### ISSUE THREAD

**Issue by gjuoun** [denjucks/denjucks#9]
Envirionment: 
- OS: windows 10
- Deno V1.1.1

Code: 

```ts
// @deno-types="https://deno.land/x/denjucks/mod.d.ts"
import denjucks from "https://deno.land/x/denjucks/mod.js";

console.log(
    denjucks.renderString("hello {{ txt }}", {txt: "world"})
);
```
Result: 
```bash
error: Failed to get compiled source code of https://deno.land/x/denjucks/src/deps/path/std/path/mod.ts.
Reason: The system cannot find the path specified. (os error 3)
```

I think it's a problem related to import path module in js file, maybe there is something to do with the path import in denjucks source files.

I will make a merge request later.

**Comment by andreespirela** [comment]
@denjucks @ogonkov In order to avoid duplicate tickets, I have mentioned my issue and how it is affecting. Thank you!

**Comment by andreespirela** [comment]
It seems like it is **only** happening in Windows. It is happening to **ALL** versions of Denjucks (I pointed out to every single version and I was getting the same error) which makes me think this is highly related to the compatibility of Denjucks with one of the latest Deno upgrades.

**Comment by gjuoun** [comment]
Fixed by @cj81499 in #12
